.golangci.yml

```diff
@@ -1,40 +1,58 @@
-issues:
-  exclude-rules:
-    - path: examples/*
-      linters:
-        - forbidigo
+version: "2"
```

Comment:
> update golangci version 

Please move it to a separate PR. I can see that some other PRs fail due to golangci issues (e.g. https://github.com/scylladb/scylla-cdc-go/pull/30), so I think it makes sense to consider it separately - and it can be merged much more quickly.